sig/handwritten/app/controllers/admin.rbs

```diff
@@ -0,0 +1,2 @@
+module Admin
+end
```

Comment:
In Rails app, zeitwerk autoloader automatically defines `Admin` module
internally.  But RBS does not support auto module definition.  Therefore
we need to add the type to the module manually.